Scripts/extras/pull_and_build_from_git.py

```diff
@@ -564,12 +564,18 @@ def build_and_install_cmake(self):
         if cmake_install_filter:
             # If there is a custom install filter, then we need to install to another temp folder and copy over based on the filter rules
             install_target_folder = self.base_temp_folder / 'working_install'
-            if not install_target_folder.is_dir():
-                install_target_folder.mkdir(parents=True)
         else:
             # Otherwise install directly to the target
             install_target_folder = self.build_install_folder
 
+        install_target_folder = install_target_folder.resolve().absolute()
+
+        if self.clean_build and install_target_folder.is_dir():
+            delete_folder(install_target_folder.absolute().resolve())
```

Comment:
do we need the absolute().resolve()if we already did that on line 571?

Reply:
The [Path.resolve](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve) already makes the path absolute